2024/2024-ssl-teams-vs-teams-passing-challenge.adoc

```diff
@@ -0,0 +1,105 @@
+:source-highlighter: highlightjs
+
+= RoboCup 2024 Team vs. Team passing Challenge
+{docdate}
+:toc:
+:stem: latexmath
+:sectnumlevels: 0
+
+// add icons from fontawesome in a up-to-date version
+ifdef::backend-html5[]
+++++
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+++++
+endif::backend-html5[]
+
+:icons: font
+:numbered:
+
+NOTE: References to the male gender in the rules with respect to referees, team
+members, officials, etc., are for simplification and apply to both males and
+females.
+
+== Goal
+
+A key distinguishing characteristic of the Small Size League is its tight integration of software and hardware, which is crucial to create a dynamic, collaborative soccer match. 
+While this goal was pursued in last year's SSL Chip Pass Challenge, the goal of this year's challenge is to utilize last year's work and adapt it to a more realistic situation.
+
+Therefore this year two teams will try to play as many passes as possible while competing against each other. To achieve this, the ruleset used last year has been adapted to the new situation.
+
+Although the teams, which have participated in one of the previous passing challenges, might have a slight advantage, we would like to encourage all teams to participate in this technicall challenge.
+
+== Procedure
+
+The technicall challenge will be held on a Div B field with a maximum of 4 robots per team.
+
+To level the playing field and to remove the impact of the performance of the opponent team on a teams score, each team will play two games against different contestants of the technical challenge.
+
+Each game will have the following procedure:
+
+* If both teams agree on the team colors, their suggestion will be used. If they do not agree, the referee will make a coin toss and the winning team will get the yellow team color. Even in this case, the teamcolor will not be changed during the half time break.
+* To start the game, a kick-off command for the yellow team will be issued from the game controller. For the start the robots can be positioned can positioned in any way prefered, along as the positioning does not violate the usual ruleset for a kick-off.
```

Comment:
Why so complicated? Can't we just say: We play with normal match rules, except ...

Reply:
I think with the new version from today, it is probably easier written than before.
But the main idea behind specifying, which team starts and now which team gets which color, I wanted to speed up the preparation for each of the games during the TC.